actions/resources.ts

```
@@ -166,54 +179,61 @@ export async function addResource(data: ResourceFormValues) {
-        coverImage: data.coverImage
-      }
-    })
+        coverImage: data.coverImage,
+      },
+    });
 
-    revalidatePath('/admin/resources')
-    return { resource }
+    revalidatePath("/admin/resources");
+    return { resource };
   } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)